### PR TITLE
WebWork

### PR DESCRIPTION
**Pull request by Marine10**

### CODE REVIEW

**Comment by rosa** [comment]
Hi @Marine10, thanks for sending this pull request! I had a look at the URL listed for this integration and the logo used belongs to Basecamp 2. Is this integration for Basecamp 3 or 2? 
Thank you!

**Comment by Marine10** [comment]
Hi,

Thank you very much for checking the request.
It is Integration for Basecamp 3. It seems that our team members put the
wrong logo.

If you could send the right one, we will change it immediately.

Best regards,
Marine

**Comment by rosa** [comment]
Thank you @Marine10! You can find the logo [here](https://basecamp.com/about/press), under the "Download the Basecamp Logo" section.

**Comment by Marine10** [comment]
Hi Rosa,

We have changed the logo if it is still showing the old one, just hard
refresh the page (Ctrl+Shift+R).

Please, let me know if there is anything else we can do.

Best regards,
Marine

File: integrations.yml [hunk 1]
```diff
@@ -390,6 +390,12 @@
   website: "https://www.jitbit.com/helpdesk/basecamp/"
   icon: "jitbit.png"
   category: 7
+-
+  name: "WebWork Time Tracker"
+  description: "Easily track your Basecamp projects and tasks, get powerful reports and boost productivity."
+  website: "https://www.webwork-tracker.com/integrations/basecamp"
+  icon: "webwork.png"
+  category: 2
 -
```
Comment (rosa) [inline integrations.yml]: Could you move the integration to be at the end of the file, under the `Centreli` one?

**Comment by rosa** [comment]
Thank you! That's perfect. I just added a small comment, when that's resolved I'll merge this.

**Comment by Marine10** [comment]
Thanks.
Can you, please, check it? I have just removed it.

Thank you in advance!

**Pull request merged**